lib/sles4sap_publiccloud.pm

```diff
@@ -728,7 +728,7 @@ sub create_playbook_section_list {
         if ($args{registration} eq 'suseconnect') {
             # For the moment, only role version of the registration playbook, using roles from community.sles-for-sap,
```

Comment:
Please remove this comment too

```
# For the moment, only role version of the registration playbook, using roles from community.sles-for-sap,
# is supporting force suseconnec.
```

Reply:
Ah, sorry, forgot... let me open a new pr